crypto/asn1/bio_asn1.c

```diff
@@ -16,6 +16,7 @@
 #include <string.h>
 #include <internal/bio.h>
 #include <openssl/asn1.h>
+#include "e_os.h"
```

Comment:
So, I'm still not clear whether any of the above comments constitute an actual objection to *this* PR, or whether it is ok to address them in a separate PR (my preferred approach)?